src/main/java/discord4j/discordjson/json/PartialMemberData.java

```diff
@@ -41,4 +41,8 @@ static ImmutablePartialMemberData.Builder builder() {
 
     @JsonProperty("communication_disabled_until")
     Possible<Optional<String>> communicationDisabledUntil();
+
+    default Long flags() {
```

Comment:
Hi! Is boxed long actually needed here?

Reply:
oh... not, i base in the User flags but in there the value is nullable.. here in teory not then better change this.